Add icons to windows and (windows) executable

ahockersten opened the pull request:
This adds a green worm to the main window and a blue worm to the spectator window. It also bundles an icon for the windows executable.

I didn't bother with a Linux icon. That's usually part of the package's job, and I haven't considered Linux packaging yet.

Should fix #6, except for the Mac. @emlai could you review this and do additional Mac changes to get the icon in?

I'm (very) open to switching icons, here I focused on getting the code correct.

Commit:
Build fixes for VS

Use different SDL and remove dependency on GVL props

Commit:
Various improvements to windows build

- Removed _CONSOLE from various non-builds
- Use better intermediate and output directories

Commit:
Add support for window icons

Commit:
Renamed file

Commit:
Add icon to windows executable

emillaine commented:
Mac doesn't actually allow per-window icons, only one icon per app instance, shared by all windows created by the app.

So for example, [this line](https://github.com/ahockersten/liero/pull/7/files#diff-845b3559f18e8dd340096359b941b57aR337) changes the icon of the _whole_ app when a spectator window is opened. Closing the spectator window doesn't change the icon back to the green one.

Should we simply `#ifdef` out the spectator window icon code on Mac and always use the green icon when on Mac, or what do you think would be best?

Commit:
Null-check for icons

Commit:
Add windows-specific icons

For some reason, large icons cannot be loaded correctly on Windows

Commit:
Work around some window icon issues with spectator

ahockersten commented:
I added the ifdef, and added special smaller icons for windows, because for some reason it was only loading one of them when they were that size. The displayed size is quite tiny in any case.
